trunk/cities/DK.php

```diff
@@ -0,0 +1,605 @@
+$cities['DK'] = array(
```

Comment:
I understand that Denmark does not have states or counties. Nevertheless, move DK.php into the states folder. I'll probably rename states in future to something more universal.

Also, ensure the file (DK.php) matches the structure and format in https://github.com/chitezh/woocommerce_states_places/blob/master/trunk/states/NG.php

Reply:
Or rather move it to `places` I'll take it from there. Ideally we will repeat Denmark in the states folder and have just one option "Denmark" as the only state